app/src/main/res/navigation/navigation_graph.xml

```diff
@@ -207,9 +207,9 @@
 
         <argument
             android:name="snackbarMessage"
+            android:defaultValue="null"
```

Comment:
umm thats what I did just used "null" word instead of " '' ' " . If  want would do that.

Reply:
I think it would help people understand the code better when they read it if the default value is something sensible, like a blank string. Using `"null"` technically works, but since it is not a standard value it might cause difficulty for people trying to understand the code later.

Reply:
For better readablity, I suppose , we should write "Not-Set", that's clear in first go